docs/questions/sharing/visualizations/line-bar-and-area-charts.md

```diff
@@ -125,22 +125,41 @@ Here you'll find additional settings for configuring your x and y axes (as in ax
 
 ### X-axis
 
-- Show label (the legend label for the axis).
+- Show label (the label for the axis).
 - Rename the axis
 - Show line and marks
 - Scale: Timeseries or Ordinal.
 
 ### Y-axis
 
-- Show label (the legend label for the axis).
+- Show label (the label for the axis).
 - Rename the axis
 - Split y-axis when necessary
 - Auto y-axis range. When not toggled on, you can set the y-axis range (it's min and max values).
 - Unpin from zero. Allows you to "Zoom in" on charts with values well above zero. Here's an example (note the y-axis starts at 20,000):
-![y-axis unpinned from zero](../../images/unpinned-from-zero-y-axis.png)
+  ![y-axis unpinned from zero](../../images/unpinned-from-zero-y-axis.png)
```

Comment:
Will Jekyll support an indented image? Workaround: have "unpin" as the last item, then show the image.

Reply:
i think this is coming from vscode "format on save".. I'll switch the order